Dockerfile

```
@@ -22,7 +22,7 @@ COPY vendor/ vendor/
 # Build
 RUN CGO_ENABLED=0 GO111MODULE=on go build -mod=vendor -a -o manager main.go
 
-FROM registry.ci.openshift.org/ocp/4.10:base
+FROM registry.access.redhat.com/ubi8/ubi
```

I understood we would use `debian`? To sync with the builder, I would do:
```suggestion
FROM debian:bullseye-slim
```

---

Yea I have already fixed that one but didn't push yet.